roles/test_report_send/doc/env2vars.md

```
@@ -0,0 +1,293 @@
+2. single environment variable, pointing at a file with structured JSON payload
+3. both
+
+Still, each CI system has its own considerations, terminology and this data is kept in different structures/environment variables.
+
+## Solution overview
+
+We not have full control of our reporting systems, since we are only users.
+So we cannot rely on our capability to post-ingest processing.
+In short, we 
+1. can't admin the reporting (regulation, policies)
+2. we do not want to maintain CI specific queries => we want to query events from any CI using the same query expressions, differing only in the specific variable values (team, release, product, project, etc.)
```

markdownlint-cli2 (0.17.2)

Line length
Expected: 120; Actual: 209

(MD013, line-length)